addons/web/static/src/views/view_button/view_button_hook.js

```diff
@@ -117,8 +117,8 @@ export function useViewButtons(ref, options = {}) {
                             ...(clickParams["confirm-label"] && {
                                 confirmLabel: clickParams["confirm-label"],
                             }),
-                            ...(clickParams["cancel-label"] && {
-                                cancelLabel: clickParams["cancel-label"],
+                            ...(clickParams["close"] && {
+                                cancelLabel: clickParams["close"],
```

Comment:
What is this doing here seems unrelated to the changes? cc @aab-odoo

Reply:
Indeed, good catch @sdegueldre 

@ppr-odoo could you remove this diff ? This should not be necessary for this task...